tools/devops/automation/build-pipeline.yml

```diff
@@ -162,6 +162,10 @@ variables:
 - template: templates/variables.yml
 - name: MicrobuildConnector
   value: 'MicroBuild Signing Task (DevDiv)'
+- name: Codeql.Enabled  # do not add the the shared variables.yml because it will force other pipelines
+  value: false  # it is a slow process, do not enable on pipelines
+- name: Codeql.DoNotStartTimes
+  value: "Mon;Tue;Wed;Thu;" # skip duing most of the week yet perfor the scan on Fridays + Weekend
```

Comment:
@rolfbjarne  @dalexsoto you input is welcome, running every week sounds ok to me.

Reply:
So we'll have to run the PR again tomorrow to see if it works?

Reply:
it is not enabled on PRs, so it will be in CI. That is a recommendation from the codeql team. I believe the process is quite heavy.

Reply:
Do you have a link to a CI run then?